resources/assets/enderio/lang/en_us.lang

```diff
@@ -111,3 +111,196 @@ enderio.gui.telepad.tank=Reagent Tank
+tile.block_attractor_obelisk.tooltip.detailed.line4=within the device
+
+tile.block_aversion_obelisk.name=Aversion Obelisk
+tile.block_aversion_obelisk.tooltip.detailed.line1=Prevents mobs from spawning
+tile.block_aversion_obelisk.tooltip.detailed.line2=within its area of effect
+
+tile.block_inhibitor_obelisk.name=Inhibitor Obelisk
+tile.block_inhibitor_obelisk.tooltip.detailed.line1=Prevents teleporting within 
+tile.block_inhibitor_obelisk.tooltip.detailed.line2=its area of effect
+
+tile.block_relocator_obelisk.name=Relocator Obelisk
+tile.block_relocator_obelisk.tooltip.detailed.line1=Relocates mobs that spawn
+tile.block_relocator_obelisk.tooltip.detailed.line2=within its area of effect
+
+tile.block_weather_obelisk.name=Weather Obelisk
+tile.block_weather_obelisk.tooltip.detailed.line1=Changes the current weather
+tile.block_weather_obelisk.tooltip.detailed.line2=Requires a lot of power and
+tile.block_weather_obelisk.tooltip.detailed.line3=a catalyst item
+
+tile.block_experience_obelisk.name=Experience Obelisk
+tile.block_experience_obelisk.tooltip.detailed.line1=Stores Experience
+tile.block_experience_obelisk.tooltip.detailed.line2=Can output liquid XP to
+tile.block_experience_obelisk.tooltip.detailed.line3=adjacent tanks
+
+; no description
+tile.block_painter.name=Painting Machine
+
+tile.block_reservoir.name=Reservoir
+tile.block_reservoir.tooltip.detailed.line1=When connected with other Reservoirs:
+tile.block_reservoir.tooltip.detailed.line2=Add two buckets of water for
+tile.block_reservoir.tooltip.detailed.line3=an infinite water source.
+tile.block_reservoir.tooltip.detailed.line4=Produces one bucket per second
+tile.block_reservoir.tooltip.detailed.line5=R-Click with a wrench to
+tile.block_reservoir.tooltip.detailed.line6=enable auto-output
+
+tile.block_sag_mill.name=SAG Mill
+tile.block_sag_mill.tooltip.detailed.line1=Grinds ore and other items
+tile.block_sag_mill.tooltip.detailed.line2=Add grinding balls to increase output
+tile.block_sag_mill.tooltip.detailed.line3=and efficiency
+tile.block_sag_mill.tooltip.detailed.line4=Upgrade with capacitors to increase
+tile.block_sag_mill.tooltip.detailed.line5=speed and energy storage
+tile.block_sag_mill.tooltipPrefix=Contains
+
+tile.block_slice_and_splice.name=Slice'N'Splice
+tile.block_slice_and_splice.tooltip.detailed.line1=Slices heads and splices in
+tile.block_slice_and_splice.tooltip.detailed.line2=all sorts of useful things
+
+tile.block_solar_panel.name=Photovoltaic Cell
+tile.block_solar_panel.advanced.name=Advanced Photovoltaic Cell
+tile.block_solar_panel.vibrant.name=Vibrant Photovoltaic Cell
+tile.block_solar_panel.tooltip.common.line1=Solar Power!
+tile.block_solar_panel.tooltip.detailed.line1=Produces power during daylight hours
+tile.block_solar_panel.tooltip.detailed.line2=Must have a clear line of sight to
+tile.block_solar_panel.tooltip.detailed.line3=the sky
+enderio.tooltip.efficiency=Efficiency
+enderio.tooltip.sunlightBlocked=Sky Blocked!
```

Comment:
These two are unused. Please doublecheck all keys that start with "enderio.", if you cannot see them in the running game, leave them behind.